webapp/channels/src/components/admin_console/ldap_wizard/ldap_helpers.tsx

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import type {MessageDescriptor} from 'react-intl';
+
+import WithTooltip from 'components/with_tooltip';
+
+import type {GeneralSettingProps} from './ldap_wizard';
+
+import SchemaText from '../schema_text';
+
+/**
+ * LDAP-specific component for rendering help text with hover tooltip
+ */
+export const LDAPHelpTextWithHover: React.FC<{
+    baseText: string | JSX.Element | MessageDescriptor;
+    baseIsMarkdown?: boolean;
+    baseTextValues?: {[key: string]: any};
+    hoverText: string | JSX.Element | MessageDescriptor;
+    hoverIsMarkdown?: boolean;
+    hoverTextValues?: {[key: string]: any};
+}> = ({baseText, baseIsMarkdown, baseTextValues, hoverText, hoverIsMarkdown, hoverTextValues}) => {
+    return (
+        <>
+            <SchemaText
+                isMarkdown={baseIsMarkdown}
+                text={baseText}
+                textValues={baseTextValues}
+            />
+            {' '}
+            <WithTooltip
+                title={(
+                    <SchemaText
+                        isMarkdown={hoverIsMarkdown}
+                        text={hoverText}
+                        textValues={hoverTextValues}
+                    />
+                )}
+            >
+                <span
+                    style={{
+                        color: '#0066cc',
+                        cursor: 'pointer',
+                        textDecoration: 'underline',
+                    }}
+                >
```

Comment:
To be accessible, this needs to be focusable. Ideally, I think this would be a button that keeps the tooltip visible after the user clicks on it, but I don't think `WithTooltip` supports that, so I'd recommend making this into a link with `href='#'` and a click handler that just calls `preventDefault`. That should also mean that we don't need to manually specify the style of it since it should just get that from being an `a` tag